domain/usecases/transfers/list_transfers.go

```diff
@@ -0,0 +1,16 @@
+package transfer
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/rodolfoalvesg/api-banking/api/domain/entities/transfers"
+)
+
+func (u UsecaseTransfers) ShowTransfers(ctx context.Context, accID string) ([]transfers.Transfer, error) {
+	TransferList, err := u.repo.ListAllTransfers(ctx, accID)
+	if err != nil {
+		return []transfers.Transfer{}, fmt.Errorf("Error showing transfers: %v", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("showing transfers: %w", err)
```
- Você pode retornar `nil` pra valores inválidos de map/slices/interfaces.
- O lance da mensagem do erro começar com maiúscula.
- Usar o `%w` em vez de `%v` possibilita que os callers dessa função acessem os "erros filhos" usando `errors.Is()`/`errors.As()`/`errors.Unwrap()` com a variável do erro, i.e., com `%w` vc da wrap na corrente de erros, com `%v` (ou `%s`) vc só concatena uma string.
- Usar "error" no começo de uma mensagem de erro pode ficar redundante quando as mensagens ficam concatenadas: "error ....: error ....: error".